support-workers/build.sbt

```diff
@@ -55,6 +55,7 @@ assemblyMergeStrategy in assembly := {
   case PathList("models", xs@_*) => MergeStrategy.discard
   case x if x.endsWith("io.netty.versions.properties") => MergeStrategy.first
   case x if x.endsWith("module-info.class") => MergeStrategy.discard
+  case "mime.types" => MergeStrategy.first
```

Comment:
this occurs in both aws sdks, both are slightly different but similar enough. It's just a mapping from extension to mime type so probably not used by anything we do anyway.